src/lib/utility.js

```diff
@@ -0,0 +1,4 @@
+export const capitalize = (s) => {
```

Comment:
Very cool. You could also use `lodash.capitalize` for this. lodash is a big lib, so best to only install the specific methods you need, but it is very handy. Up to you. https://lodash.com/docs/4.17.15

Reply:
Oh cool! Yea maybe that'd be the best way to go. Since it's a big library, would you say we go for lodash if there are other methods we're looking to use from it as well?

Reply:
It's what I would do, but only install `npm i --save lodash.capitalize` rather than all of lodash. Although I did have a question about why capitalize the user's input at all. I think it's best not to make assumptions about what they'd want, but maybe you all have a different reason.

Reply:
That's fair! I didn't realize you could just install a particular method of a library, good to know. I was just doing the capitalize for visual consistency, both on the form field and the list view) but was wondering the exact same thing about making assumptions about the user's intentions. So I'll remove it!